### PR TITLE
DDF-2480 Fixed Updload dialog to reopen if closed during an upload

### PR DESCRIPTION
**Pull request by timothytierney**
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@andrewkfiedler
@jrnorth
@azgoalie
@emanns95 
@ahoffer
@NGoss
@spearskw

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@stustison 
@jlcsmith

#### How should this be tested?
Open the file upload dialog and select two large files to upload. While the uploads are in progress, close the dialog using the x. Once the dialog has been closed, it cannot be re-opened.

#### Any background context you want to provide?

This was caused by a change made in DDF-1643

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by emanns95** [comment]
Confirmed works with IE10, Chrome, Firefox 👍

File: catalog/ui/search-ui/standard/src/main/webapp/js/view/ingest/IngestModal.view.js [hunk 1]
```diff
@@ -37,8 +37,6 @@ define([
             },
             initialize: function () {
                 // there is no automatic chaining of initialize.
-                Modal.prototype.initialize.apply(this, arguments);
```
Comment (stustison) [inline catalog/ui/search-ui/standard/src/main/webapp/js/view/ingest/IngestModal.view.js]: We can't do this here. Removing this line causes the view to not be cleaned up. But we create a new modal in the IngestMenu.js file, which then causes this view to become a zombie. I think we need to look at the `isUnfinished` function and figure out why that isn't keeping the original modal around.

Comment (timothytierney) [inline catalog/ui/search-ui/standard/src/main/webapp/js/view/ingest/IngestModal.view.js]: The change introduced in DDF-1643 added in a view.destroy() in the modal.js initialize function  when the model is hidden, so it destroy's the view which caused this issue.

**Comment by emanns95** [comment]
Another PR for this issue has been opened: https://github.com/codice/ddf/pull/1281

**Pull request closed without merging**